kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerControllerImpl.java

```diff
@@ -189,7 +185,6 @@ public KieServerSetup update(KieServerStateInfo kieServerStateInfo) {
 
         // we update and notify
         notificationService.notify(new ServerInstanceUpdated(serverInstance));
-        notificationService.notify(new ServerTemplateUpdated(serverTemplate));
```

Comment:
Why is this notification removed?

Reply:
the update does not change the server template (only can happen from BC)